src/Features/CSharp/Portable/UseExpressionBodyForLambda/Fixing.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseExpressionBodyForLambda
+{
+    internal partial class UseExpressionBodyForLambdaCodeStyleProvider
+    {
+        protected override Task<ImmutableArray<CodeAction>> ComputeCodeActionsAsync(Document document, Diagnostic diagnostic, CancellationToken cancellationToken)
+        {
+            var codeAction = new MyCodeAction(
+                diagnostic.GetMessage() + "-Fix",
+                c => FixWithSyntaxEditorAsync(document, diagnostic, c));
```

Comment:
this -Fix suffix is simply for ensuring that the right codepath was generating the code action when testing this in VS.